.github/workflows/release.yml

```diff
@@ -35,14 +35,11 @@ jobs:
         run: |
           echo "🍻" > release.txt
       - name: Release
-        uses: softprops/action-gh-release@v1
         if: startsWith(github.ref, 'refs/tags/')
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          prerelease: true
-          body_path: release.txt
-          files: ${{ steps.info.outputs.artifacts_dir }}/*
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          gh release upload ${{ github.ref_name }} ${{ steps.info.outputs.artifacts_dir }}/*
```

Comment:
Should we make it a prerelease to align with the original code?

Reply:
I think we don't need to do it. It also upload tarball when we create a tag like this test https://github.com/Yu-Jack/docker-machine-driver-harvester/releases.

Reply:
@FrankYang0529 do we need prerelease set to true?

Reply:
Some of previous releases were created by github-actions. Probably, it's better to create a draft release first and manually change to latest release.

![Screenshot 2024-12-18 at 4 03 11 PM](https://github.com/user-attachments/assets/c0209adf-3d3c-442d-ba79-9054d8994e92)

Reply:
It depends on tag or release. If we prefer to create a tag first, then we can use draft release. If we prefer to create a release directly, then we don't need to create draft relase.